lib/stanford/storage_object_validator.rb

```diff
@@ -0,0 +1,35 @@
+require 'moab'
+
+module Stanford
+  # druids are Stanford specific entities
+  class StorageObjectValidator < Moab::StorageObjectValidator
+
+    # TODO: test to make sure constants don't collide on underlying int vals?
+    # keep from stepping on previously defined error code constants.
+    DRUID_MISMATCH = superclass.error_code_to_messages.keys.max + 1
+
+    def validation_errors
+      errors = []
+      errors.concat super
+      errors << identify_druid
+      errors.flatten
```

Comment:
i think the idea of using `.concat` for multiple values is to avoid having to use flatten.  But no biggie.

Reply:
i would also vote for `errors.concat identify_druid`.  if only because we're otherwise consistent about that elsewhere (but it also just reads a little nicer as one step, imho).